internal/cli/atlas/livemigration/link/create.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package link
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var createTemplate = "Link Token '{{.LinkToken}}' successfully created.\n"
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store        store.LinkTokenCreator
+	accessListIP []string
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+func (opts *CreateOpts) Run() error {
+	createRequest := opts.newTokenCreateRequest()
+
+	r, err := opts.store.CreateLinkToken(opts.OrgID, createRequest)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newTokenCreateRequest() *mongodbatlas.TokenCreateRequest {
+	return &mongodbatlas.TokenCreateRequest{
+		AccessListIPs: opts.accessListIP,
+	}
+}
+
+// mongocli atlas liveMigration|lm link create [--accessListIp accessListIp] [--orgId orgId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create one new link-token.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.initStore,
```

Comment:
no need to mark the flag required, org can be set via the profile or an env variable so we use this instead
```suggestion
				opts.ValidateOrgID,
				opts.initStore,
```